### ISSUE TITLE
Difference between sdlx and sd15

### ISSUE THREAD

**Issue by kulikovv** [huaishengzhu/DSPO#3]
Hi! 

Could you please explain the difference between losses for sd15 and sdxl, why model_diff_w is not taken into count in sdxl scenario:
```python 
if args.sdxl:
    if pred2.pow(2).mean(dim=[1,2,3]).mean() >= 0.1:
        loss = ((1 - F.sigmoid(inside_term)[:, None, None, None]) * pred2).pow(2).mean(dim=[1,2,3]).mean()
    else:
        loss =  (1 - F.sigmoid(inside_term)[:, None, None, None]).pow(2).mean(dim=[1,2,3]).mean()
else:
    loss = (model_diff_w - args.beta_dpo * (1 - F.sigmoid(inside_term)[:, None, None, None]) * pred2).pow(2).mean(dim=[1,2,3]).mean()
```
the question is about **beta_dpo** for sdxl it is large 1000-5000 (like in the original DPO paper), but for sd15 you recommend 0.01 and can severely affect the inside_term.

Also in the paper lamba inside the step-wise reward function is the same that in the final loss. But in the code lamba before the step-wise reward function (inside_term) is -0.5*beta_dpo, and in the loss it goes with just beta_dpo. The error is in the paper or in the code?

**Comment by kulikovv** [comment]
Thank you for the explanation!
